CZI.md

```diff
@@ -107,73 +82,64 @@ the graduate student who is leading the this effort as her thesis work.
-"fun" which is awkward to rely on solely volunteer effort to
-accomplish in a timely manner.  In addition to the inherent value of
-completing these tasks, having them promptly and reliably addressed
-improves the contribution experience for everyone working on the
-project.  We propose to devote three quarters of a developers time to
-handling these tasks.
+individually are small, but are frequently reactive, time critical, and 
+often tedious. They are best handled by a paid developer because it is 
+awkward to rely on solely volunteer effort to accomplish tasks that they often 
+do not have intrinsic interest in carrying out. 
+
+In addition to the inherent value of completing these tasks, having them 
+promptly and reliably addressed improves the contribution experience for 
+everyone working on the project.  We propose to devote three quarters of a 
+developers time to handling these tasks.
 
 In addition to the reactive work there are substantial but incremental
 improvements that will benefit from long blocks of dedicated work.
 Examples of this work include fixing long standing rendering and
 performance issues, deep-dive documentation, homogenizing and
 smoothing the API, and new user-facing functionality. For example, we
-have a long-standing prototype adding a color-blind simulation filter
-to the UI.  To turn this prototype into a user-ready functionality we
+have a long-standing prototype for adding a color-blind simulation filter
+to the UI.  To turn this prototype into user-ready functionality, we
 need to do additional research to make sure we are implementing the
 filters accurately in addition to the engineering effort.  We propose
 to devote about a quarter of a developer to this effort which should
-enable us to complete 5 of these projects.  We will work with
-downstream bio-projects to identify what will have the greatest
-impact.
-
-As a library Matplotlib needs to be able to support the use cases for
-the full range of science domains and complex novel visualization [1]
-that provide new insight.  At the same time the most common
-visualizations in a domain need to be fluid for the end-practitioners,
-with the "obvious" customization options exposed.  To achieve both of
-these goals we need to continue to foster a two layer eco-system with
+enable us to complete 5 of these projects.  
+<!--confused about 5 'cause you only mentioned this color blind thing that feels random-->
+We will work with downstream bio-projects to identify what will have the greatest
+impact. <!--why? and then I think most of your grant time ends up spent on downstream libraries 'cause 3 points of contact - me & Elliot & you - sounds like it'll get messy fast--->
+
+As a library Matplotlib needs to support use cases for
+the full range <!--not vast range?--> of science domains and complex novel visualizations [1]. 
+At the same time the most common visualizations in a domain need to be fluid 
+<!--what does fluid mean?--> for the end-practitioners,
```

Comment:
Easy and not getting in the way.  This is my understanding of why people who do lots of stats like ggplot because it is tuned to their data structures and the standard visualizations.